install/kubernetes/helm/istio/charts/pilot/templates/config.yaml

```diff
@@ -0,0 +1,28 @@
+{{ define "config.yaml.tpl" }}
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: istio-policy
+  namespace: {{ .Release.Namespace }}
+spec:
+  name: istio-policy.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
@rshriram is this how it is supposed to work? how do we verify this takes effect without stressing the system?

Reply:
Set the maxRequests and maxReq/conn values to absurdly low numbers so that even moderate stress will cause the same issue.

Since this is simply restoring the config that has always existed, I don't think we need to do the above test.

Reply:
Ok, once @rshriram or @costinm confirm the semantics of this API, we should be OK to merge.

Reply:
As mandar says. Also you can dump proxy config (/debug/adsz) to see it it shows up.

Reply:
You should also set mTLS here if using control plane auth.

Reply:
yes it does